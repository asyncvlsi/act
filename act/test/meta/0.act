pint x = 3+5;

preal y = 3.0/x;

bool a, b;

prs {
 (i:x: a<i+y>-> b-)
}
