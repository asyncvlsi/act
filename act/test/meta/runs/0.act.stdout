/* instances */
bool b;
bool a;
prs {
a<0.375> -> b-
a<1.375> -> b-
a<2.375> -> b-
a<3.375> -> b-
a<4.375> -> b-
a<5.375> -> b-
a<6.375> -> b-
a<7.375> -> b-
}
